core/amber/src/main/scala/Engine/Common/tuple/Tuple.scala

```diff
@@ -1,8 +1,9 @@
-package Engine.Common.AmberTuple
+package Engine.Common.tuple
 
 import java.util
 
 import Engine.Common.AmberField.FieldType
+import Engine.Common.tuple.amber.AmberTuple
 
 import collection.JavaConverters._
 import scala.util.hashing.MurmurHash3
```

Comment:
This comment is for the `object Tuple` below. Why do we need a singleton class for Tuple?

Reply:
It's the helper functions for to construct an AmberTuple, so instead of `new AmberTuple(data)` the code does `Tuple.from(data)` We could remove them later.

Reply:
Also, the methods from line 33-40 should use the same definition of getAnyValAs()